Rx/v2/test/operators/merge_delay_error.cpp

```diff
@@ -0,0 +1,390 @@
+                .merge_delay_error(
+                    so,
+                    rxs::range(sectionCount, (sectionCount * 2) - 1, 1, so),
+                    rxs::range(sectionCount * 2, onnextcalls - 1, 1, so))
+                .as_blocking()
+                .count();
+
+            auto finish = clock::now();
+            auto msElapsed = duration_cast<milliseconds>(finish.time_since_epoch()) -
+                   duration_cast<milliseconds>(start.time_since_epoch());
+            std::cout << "merge sync ranges : " << n << " subscribed, " << c << " emitted, " << msElapsed.count() << "ms elapsed " << std::endl;
+        }
+    }
+}
+
+SCENARIO("observe_on merge ranges", "[hide][range][observe_on][merge][perf]"){
+    const int& onnextcalls = static_onnextcalls;
+    GIVEN("some ranges"){
+        WHEN("generating ints"){
+            using namespace std::chrono;
+            typedef steady_clock clock;
+
+            auto so = rx::observe_on_event_loop();
+
+            int n = 1;
+            auto sectionCount = onnextcalls / 3;
+            auto start = clock::now();
+            int c = rxs::range(0, sectionCount - 1, 1, so)
+                .merge_delay_error(
+                    so,
+                    rxs::range(sectionCount, (sectionCount * 2) - 1, 1, so),
+                    rxs::range(sectionCount * 2, onnextcalls - 1, 1, so))
+                .as_blocking()
+                .count();
+
+            auto finish = clock::now();
+            auto msElapsed = duration_cast<milliseconds>(finish.time_since_epoch()) -
+                   duration_cast<milliseconds>(start.time_since_epoch());
+            std::cout << "merge observe_on ranges : " << n << " subscribed, " << c << " emitted, " << msElapsed.count() << "ms elapsed " << std::endl;
+        }
+    }
+}
+
+SCENARIO("serialize merge ranges", "[hide][range][serialize][merge][perf]"){
+    const int& onnextcalls = static_onnextcalls;
+    GIVEN("some ranges"){
+        WHEN("generating ints"){
+            using namespace std::chrono;
+            typedef steady_clock clock;
+
+            auto so = rx::serialize_event_loop();
+
+            int n = 1;
+            auto sectionCount = onnextcalls / 3;
+            auto start = clock::now();
+            int c = rxs::range(0, sectionCount - 1, 1, so)
+                .merge_delay_error(
+                    so,
+                    rxs::range(sectionCount, (sectionCount * 2) - 1, 1, so),
+                    rxs::range(sectionCount * 2, onnextcalls - 1, 1, so))
+                .as_blocking()
+                .count();
+
+            auto finish = clock::now();
+            auto msElapsed = duration_cast<milliseconds>(finish.time_since_epoch()) -
+                   duration_cast<milliseconds>(start.time_since_epoch());
+            std::cout << "merge serial ranges : " << n << " subscribed, " << c << " emitted, " << msElapsed.count() << "ms elapsed " << std::endl;
+        }
+    }
+}
+
```

Comment:
it is not required to keep the [perf] tagged tests above, since the error handling perf is not being tested.